### ISSUE TITLE
Pywb requires restart in order to know about additions to an collections archive

### ISSUE THREAD

**Issue by N0taN3rd** [webrecorder/pywb#196]
In order to have pywb recognize that a new warc file has been added (including index.cdxj update) it must be restarted.
Is there an configuration that allows pywb to automatically know about such additions or an endpoint that can be configured to allow such functionality?

Use case: 
User initiated a crawl via Heritrix through WAIL and it finishes. WAIl then utilizes wb-manager to add the produced warc to the collection (reindexing of  the collection happens).
WAIL then informs the user their collection has been updated, the user then queries pywb through WAIL about the addition and receives a reply of not found.

@machawk1

**Comment by machawk1** [comment]
Is this still the case, @N0taN3rd? If so, @ikreymer, any advice in pywb recognizing new collections w/o requiring a restart?

**Comment by ikreymer** [comment]
Currently, yes, though am working on a new version where it will pick up collections dynamically. This is part of the new aggregator based approach. I'll let you know when its ready for testing!

**Comment by N0taN3rd** [comment]
@machawk1 yes 
@ikreymer looking forward to trying out that feature and is there any plans on allowing the creation of collections through pywb that have spaces in them? for example wailsAwesomeCollection -> wails awesome collection?

**Comment by ikreymer** [comment]
If you are interested in trying the very very latest, you can run the `new-wayback` on the `new-pywb` branch.  This is using a brand new system, which will pick up new collections added, eg.
```
new-wayback
wb-manager init coll
wb-manger add coll /path/to/warc.gz
curl "localhost:8080/coll/mp_/<url>"
```

It should support spaces as well, though I believe for now there is a validation check that prevents it.
Anyway this will continue to evolve and there's some refactoring ahead, but in general should be possible now.

**Comment by N0taN3rd** [comment]
@ikreymer awesome! thanks for letting us know :+1:

**Comment by N0taN3rd** [comment]
@ikreymer does the new-wayback pull in the [index](https://github.com/ikreymer/pywb/blob/master/pywb/templates/index.html) template anymore.
Looking at how the new-wayback works putting a config.yaml file in the collections directory pointing to it did not seem to cause this to happen.

**Comment by ikreymer** [comment]
Now available in pywb 2.0.0
